charts/opensearch/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ---
 ## [Unreleased]
 ### Added
+- Updated StatefulSet with node_dn.yml and whitelist.yml
```

Comment:
Do you mind seeing the PR(s) before on updating the CHANGELOG?

Reply:
Sure, Do you want me to remove this from the current PR and submit a different one?

Reply:
Hey @phijojo please consider this as an example
https://github.com/opensearch-project/helm-charts/pull/250/files#diff-96471865d01149e108175156796718f7584b7d5fae730d0cb5dbd98426bcd801